ReportingServicesTools/Functions/CatalogItems/Remove-RsSubscription.ps1

```diff
@@ -0,0 +1,101 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Remove-RsSubscription
+{
+    <#
+        .SYNOPSIS
+            This function removes an subscription from the report.
+        
+        .DESCRIPTION
+            This function removes an subscription from the report.
+        
+        .PARAMETER Subscription
```

Comment:
Added help to SubscriptionId parameter.

Yes, I think we should have documentation for it. 
For example, people can use `Get-RsSubscritpion` and fill a `$subscriptions` variable to work and after that use this variable (with one or more subscriptions) on `Set-RsSubscription` command like `Set-RsSubscription -Subscription $subscriptions`